crowdin.yml

```
@@ -0,0 +1,100 @@
+#
+# Your Crowdin credentials
+#
+"project_id_env" : "CROWDIN_PROJECT_ID"
+"api_token_env" : "CROWDIN_PERSONAL_TOKEN"
+"base_path" : "."
+"base_url" : "https://api.crowdin.com"
+
+#
+# Choose file structure in Crowdin
+# e.g. true or false
+#
+"preserve_hierarchy": true
+#
+# Files configuration
+#
+files: [
+  {
+     "source" : "/services/src/main/resources/locale/addon/Gamification_en.properties",
+
+    "translation" : "%original_path%/%file_name%!_%locale_with_underscore%.%file_extension%",
+    "translation_replace" : {
+      "_en!":"",
+      "de_DE": "de",
+      "en_US": "en",
+      "es_ES": "es_ES",
+      "fr_FR": "fr",
+      "it_IT": "it",
+      "pt_BR": "pt_BR",
```

In the translation.zip file, we have much more languages than theses one.
I think we must put all in this list to make it work.
```suggestion
      "ar_SA":"ar",
      "ar_OM":"aro",
      "az_AZ":"az",
      ....
      "de_DE": "de",
      "en_US": "en",
      "es_ES": "es_ES",
      "fr_FR": "fr",
      "it_IT": "it",
      "pt_BR": "pt_BR",
```
I put just the main ones. 
As this is only for download action, it is possible to test this specific action : 
launch it, but not merge. So we can confirm that in the created merge branch, all languages are ok, and fix if necessary
Should we merge before test ? or can we test before merge ?

---

I use this page to know the main regional parameter for each languages :+1: https://help.sap.com/docs/SAP_BUSINESSOBJECTS_BUSINESS_INTELLIGENCE_PLATFORM/09382741061c40a989fae01e61d54202/46758c5e6e041014910aba7db0e91070.html

---

the merge should be done in order to apply the github actions. I may comment the merge part in the begining

---

Ok, So my proposition is to do that : 
- comment merge part
- Check download part to answer this question : does all languages are ok with this configuration, or should we add all languages codes in crowdin.yml ?
- If ok, uncomment merge part and deactivate jenkins sybc/merge for gamification

---

done